scripts/class_fellow.py

```diff
@@ -1,5 +1,31 @@
 from scripts.class_person import *
+'''
+
+This fellow class inherits from Person.
+Simple, it only enables fellows get living spaces
+since staff should not have living spaces.
+
+'''
 
 
 class Fellow(Person):
-    pass
+    def __init__(
+            self, first_name,
+            second_name, person_type,
+            living_space=None):
+
+        super().__init__(first_name, second_name, person_type)
+        self.first_name = first_name
+        self.second_name = second_name
+        self.person_type = person_type
+        self.living_space = living_space
+        self.availble_living_spaces = ['Python', 'Ruby', 'JavaScript', 'Java']
```

Comment:
Rooms should not be added inside the fellow class because a fellow has no attribute `availble_living_spaces`.